neofetch

```diff
@@ -28,7 +28,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-version=7.1.0
+version=7.1.1
```

Comment:
Please don't change the version here. Otherwise everything looks good!

Reply:
Thanks, I'll take it into account next time